feat(tailwindlabs/tailwindcss): scaffold tailwindlabs/tailwindcss

Igonato opened the pull request:
[tailwindlabs/tailwindcss](https://github.com/tailwindlabs/tailwindcss) - A utility-first CSS framework for rapid UI development

## Check List

<!-- Please check the list. Please don't remove the check list. -->

- [x] Read [CONTRIBUTING.md](https://aquaproj.github.io/docs/products/aqua-registry/contributing)
  - [x] Read [OSS Contribution Guide](https://github.com/suzuki-shunsuke/oss-contribution-guide/blob/main/README.md)
- [x] [All commits are signed](https://github.com/suzuki-shunsuke/oss-contribution-guide/blob/main/docs/commit-signing.md)
  - This repository enables `Require signed commits`, so all commits must be signed
- [x] [Avoid force push](https://github.com/suzuki-shunsuke/oss-contribution-guide?tab=readme-ov-file#dont-do-force-pushes-after-opening-pull-requests)
- [x] Install and execute the package and confirm if the package works well
- [x] [Execute `cmdx s` to scaffold code](https://aquaproj.github.io/docs/products/aqua-registry/contributing/#use-cmdx-s-definitely)

<!-- Please write the description here -->

Adding support for https://github.com/tailwindlabs/tailwindcss/releases/ 

Blogpost: https://tailwindcss.com/blog/standalone-cli

Using binary release would allow developers to use tailwindcss without having to install node.


> linux: use the asset for not gnu but musl if both of them are supported

This release has both, but the `musl` part is at the end of the asset name, so 

```yaml
replacements:
  linux: linux-musl
```

like i've seen in other packages, wouldn't work. Should I add `{{.Format}}` at the end and override that? Or is there a less hacky way of fixing this?

Igonato committed:
feat(tailwindlabs/tailwindcss): scaffold tailwindlabs/tailwindcss

suzuki-shunsuke commented:
Thank you for your contribution!

> like i've seen in other packages, wouldn't work. Should I add {{.Format}} at the end and override that? Or is there a less hacky way of fixing this?

For instance, in case of [tailwindcss-linux-arm64-musl](https://github.com/tailwindlabs/tailwindcss/releases/download/v4.1.3/tailwindcss-linux-arm64-musl),

```yaml
overrides:
  - goos: linux
    asset: tailwindcss-{{.OS}}-{{.Arch}}-musl
```

suzuki-shunsuke committed:
fix(tailwindlabs/tailwindcss): use musl-based Linux builds

https://github.com/tailwindlabs/tailwindcss/releases/tag/v4.0.0-beta.9

> Add musl-based Linux builds of the standalone CLI

suzuki-shunsuke commented:
🎉 https://github.com/aquaproj/aqua-registry/releases/tag/v4.345.0
